README.md

```diff
@@ -7,7 +7,7 @@ python3 -m pip install git+https://github.com/vidursatija/dataset-dog-python.git
 ```
 
 ## Usage
-```python
+```python3
```

Comment:
this doesn't work

Reply:
it does work - try edit on the PR description, it has python3